src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -71,7 +66,7 @@ class HttpEventSSEHandle(
 
   override def sendEvent(event: MarathonEvent): Unit = {
     if (subscribed(event.eventType)) {
-      val payload = if (useLightWeightEvents) event.lightJsonString else event.fullJsonString
+      val payload = Json.stringify(eventToJson(event))
```

Comment:
I think this should be `event.jsonString` so that we do not serialize each time.